### PR TITLE
PYSDK - 30 - Lint with flake8

### LINKED ISSUE #30: Lint with flake8
**ENHANCEMENT**
1. Describe the enhancement you are requesting. Enhancements include:
   - Linting with flake8
   - code refactor

2. Should-have.

### PR DESCRIPTION
**Pull request by 0x6f736f646f**
Signed-off-by: 0x6f736f646f <blackd0t@protonmail.com>

### What does this do?
It lints all the library files to the flake8 standard

### Which issue(s) does this PR fix/relate to?
Resolves #30  

### List any changes that modify/break current functionality
No

### Have you included tests for your changes?
No.

### Did you document any new/modified functionality?
No.

### CODE REVIEW

File: lib/users.py [hunk 1]
```diff
@@ -27,51 +28,66 @@ def login(self, user):
...
+            mf_resp.error.message = errors.handle_error(
+                errors.users["update"], http_resp.status_code)
         return mf_resp
 
     def update_password(self, old_password, password, token):
         '''Changes user password'''
         payload = {
-          "old_password": old_password,
-          "password": password
+            "old_password": old_password,
```
Comment (drasko) [inline lib/users.py:80]: is this an empty space? Maybe tab would be more appropriate?

**Review by drasko — CHANGES_REQUESTED** [review]
Nice cleaning

File: lib/users.py (hunk 1 above)
Comment (0x6f736f646f) [reply lib/users.py:80]: It is a tab on my end

Comment (drasko) [reply lib/users.py:80]: Possible, it is probably difference with the previous line. Accepted.

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**